lib/schemes/local.js

```diff
@@ -23,7 +23,17 @@ export default class LocalScheme {
   mounted () {
     if (this.options.tokenRequired) {
       const token = this.$auth.syncToken(this.name)
-      this._setToken(token)
+
+      if (token) {
+        // Set logged in status
+        this.$auth.setLoggedIn(true)
```

Comment:
This sets `loggedIn` to `true` before the user is fetched. Regardless of `user` endpoint is enabled or not.